### ISSUE TITLE
Integrate CollateX with eXist XML database

### ISSUE THREAD

**Issue by gremid** [interedition/collatex#2]
Best of both worlds ...

**Comment by rhdekker** [comment]
Leif Joran Olsson is working on this and has a working prototype.

**Comment by ljo** [comment]
Yes, the xar app very close yo the public release now after some closed groups testing the last year.
